pyface/ui/qt4/tasks/dock_pane.py

```diff
@@ -61,10 +61,10 @@ def create(self, parent):
         control.setObjectName(self.task.id + ":" + self.id)
 
         # Configure the dock widget according to the DockPane settings.
-        self._set_dock_features()
-        self._set_dock_title()
-        self._set_floating()
-        self._set_visible()
+        self._set_dock_features(None)
```

Comment:
When calling the change handlers independently (without a change actually occurring) they now need to take an event argument.  If it is unused, as it is for these handlers I simply passed in `None`.

Reply:
Sorry for snooping... don't know if you have considered defining `_set_dock_features` like this?
```
def _set_dock_features(self, _=None):
    ...
```
Then the signature of the function is more explicit about that fact that the argument is not used.

Reply:
That makes more sense, thanks!